xml/leftovers.xml

```diff
@@ -87,3 +87,74 @@ skip=$((`blockdev --getsz /dev/sdX` - 33)) | hexdump -C
+    <step>
+     <para>
+      Create a realm. Replace <replaceable>NAME</replaceable> with the realm
+      name.
+     </para>
+<screen>&prompt.cephuser;radosgw-admin realm create --rgw-realm=<replaceable>NAME</replaceable> --default</screen>
+    </step>
+    <step>
+     <para>
+      Create a system user. Replace <replaceable>USER-ID</replaceable> with the
+      username. Replace <replaceable>DISPLAY-NAME</replaceable> with a display
+      name. Only the display name may contain spaces.
+     </para>
+<screen>&prompt.cephuser;radosgw-admin user create --uid=<replaceable>USER-ID</replaceable> --display-name="<replaceable>DISPLAY-NAME</replaceable>"\
+                            --access-key=<replaceable>ACCESS-KEY</replaceable> --secret=<replaceable>SECRET-KEY</replaceable> --system</screen>
+    </step>
+    <step>
+     <para>
+      Rename the default zone and &zgroup;. Replace
+      <replaceable>NAME</replaceable> with the &zgroup; or zone name.
+     </para>
+<screen>&prompt.cephuser;radosgw-admin zonegroup rename --rgw-zonegroup default --zonegroup-new-name=<replaceable>NAME</replaceable>
+&prompt.cephuser;radosgw-admin zone rename --rgw-zone default --zone-new-name us-east-1 --rgw-zonegroup=<replaceable>NAME</replaceable></screen>
+    </step>
+    <step>
+     <para>
+      Configure the master &zgroup;. Replace <replaceable>NAME</replaceable>
+      with the realm or &zgroup; name. Replace <replaceable>FQDN</replaceable>
+      with the fully qualified domain name(s) in the &zgroup;.
+     </para>
+<screen>&prompt.cephuser;radosgw-admin zonegroup modify --rgw-realm=<replaceable>NAME</replaceable> --rgw-zonegroup=<replaceable>NAME</replaceable> --endpoints http://<replaceable>FQDN</replaceable>:80 --master --default</screen>
+    </step>
+    <step>
+     <para>
+      Configure the master zone. Replace <replaceable>NAME</replaceable> with
+      the realm, &zgroup; or zone name. Replace <replaceable>FQDN</replaceable>
+      with the fully qualified domain name(s) in the &zgroup;.
+     </para>
+<screen>&prompt.cephuser;radosgw-admin zone modify --rgw-realm=<replaceable>NAME</replaceable> --rgw-zonegroup=<replaceable>NAME</replaceable> \
+                            --rgw-zone=<replaceable>NAME</replaceable> --endpoints http://<replaceable>FQDN</replaceable>:80 \
+                            --access-key=<replaceable>ACCESS-KEY</replaceable> --secret=<replaceable>SECRET-KEY</replaceable> \
+                            --master --default</screen>
+    </step>
+    <step>
+     <para>
+      Commit the updated configuration.
+     </para>
+<screen>&prompt.cephuser;radosgw-admin period update --commit</screen>
+    </step>
+    <step>
+     <para>
+      Restart the &cogw;.
+     </para>
+<screen>&prompt.cephuser.ogw;systemctl restart ceph-radosgw@rgw.`hostname -s`</screen>
+    </step>
+   </procedure>
+   <para>
+    After completing this procedure, configure a secondary zone to create a
+    secondary zone in the master &zgroup;.
```

Comment:
This sentence seems redundant. Configure a secondary zone to create a secondary zone?